parser/lexer.go

```diff
@@ -182,6 +182,7 @@ func (s *Scanner) getNextToken() int {
 // return invalid tells parser that scanner meets illegal character.
 func (s *Scanner) Lex(v *yySymType) int {
 	tok, pos, lit := s.scan()
+	lit = s.tryDecodeToUTF8String(lit)
```

Comment:
Maybe we could remove this `decode` later. It's a little bit overlaps the `decode` in peek() function